llms/mlx_lm/tuner/utils.py

```diff
@@ -119,6 +119,16 @@ def to_lora(layer):
         keys = set(["mixer.Wqkv", "moe.gate"])
     elif model.model_type == "dbrx":
         keys = set(["norm_attn_norm.attn.Wqkv", "ffn.router.layer"])
+    elif model.model_type == "internlm2":
+        keys = set(
+            [
+                "attention.wqkv",
+                "attention.wo",
+                "feed_forward.w1",
+                "feed_forward.w2",
+                "feed_forward.w3",
```

Comment:
Let's default include just the attention projections since that's what the other models do. One can specify the full set of keys in the yaml config to override this behavior.

Reply:
Sure, thanks for your comment.

Reply:
I edited it myself actually since I was testing the model locally.